client/app/components/accounts/wizard/creation.cjsx

```diff
@@ -0,0 +1,186 @@
+{IMAP_OPTIONS} = require '../../../constants/defaults'
+{Requests
+RequestStatus
+OAuthDomains} = require '../../../constants/app_constants'
+
+_     = require 'underscore'
+React = require 'react'
+
+Form    = require '../../basics/form'
+Servers = require '../servers'
+
+RequestsInFlightStore = require '../../../stores/requests_in_flight_store'
+
+RouterGetter = require '../../../getters/router'
+
+AccountActionCreator = require '../../../actions/account_action_creator'
+
+StoreWatchMixin = require '../../../mixins/store_watch_mixin'
+AccountMixin    = require '../../../mixins/account_mixin'
+
+
+ALERTS =
+    'DISCOVER_FAILED': 'DISCOVER_FAILED'
+    'CHECK_FAILED':    'CHECK_FAILED'
+    'CREATE_FAILED':   'CREATE_FAILED'
+
+
+module.exports = AccountWizardCreation = React.createClass
+
+    displayName: 'AccountWizardCreation'
+
+    mixins: [
+        StoreWatchMixin [RequestsInFlightStore]
+        AccountMixin
+    ]
+
+
+    # Build state form RequestsInFlightStore:
```

Comment:
@m4dz : to update all application, you should listen to `routerStore`instead. I still don't think that you should create a store to handle success/errors : use `routerStore` for that stuff (because its all about URL stuff)